pulp_2_tests/tests/rpm/api_v2/test_copy.py

```diff
@@ -390,6 +391,37 @@ def test_norecursive_noconservative_nodependency(self):
         ]
         self.assertEqual(len(dst_unit_ids), 1, dst_unit_ids)
 
+    def test_recursive_noconservative_dependency(self):
+        """Recursive, non-conservative, and ``walrus-0.71`` on B.
+
+        Do the following:
+
+        1. Copy ``chimpanzee`` RPM package from repository A to B using:
+           ``recursive`` as True, ``recursive_conservative`` as False, and an
+           older version of walrus package is present on the repo B before
+           the copy.
+        2. Assert that total number of RPM of units copied is equal to ``6``,
+           and the walrus package version is equal to both ``5.21`` and
+           ``0.71``.
+
+        Additional permutation added as ``--recursive`` should ensure
+        the ``latest`` version of the RPM is also copied.
+        """
+        repo = self.copy_units(True, False, True)
+        versions = [
+            unit['metadata']['version']
+            for unit in search_units(self.cfg, repo, {'type_ids': ['rpm']})
+            if unit['metadata']['name'] == 'walrus'
+        ]
+        self.assertEqual(len(versions), 2, versions)
+        self.assertEqual(versions[0], '5.21', versions)
+        self.assertEqual(versions[1], '0.71', versions)
+        dst_unit_ids = [
+            unit['metadata']['name']
+            for unit in search_units(self.cfg, repo, {'type_ids': ['rpm']})
+        ]
+        self.assertEqual(len(dst_unit_ids), 6, dst_unit_ids)
```

Comment:
What about we add those values used in assertions as constants? We can evaluate the constants that we already in place, and modified as needed.

Reply:
ack. I didn't find anything already sitting in constants that would be easy to use for all cases.

Will take more of a look later and see if anything pre-existing uses a whole data structure (don't want to break anything else).

Otherwise, I will create a simple dict for the values being used for RPM tests and reuse the same logic on applicable advanced copy updates.